setup.py

```diff
@@ -3,8 +3,8 @@
 from setuptools import setup, find_packages
 
 install_deps = [
-    'ansible-vault==1.1.1',
-    'ansible~=2.9.7',
+    'ansible-vault==2.1.0',
+    'ansible>=2.9.7,<4.3.0',
```

Comment:
Since this PR forces an upgrade, I suggest changing this to say
```suggestion
    'ansible~=4.3',
```